File: packages/@simpli/cli-scaffold/generator/template/src/model/response/UserResp.ts
/**
 * UserResp
 * @author martinlabs CRUD generator
 */
import {ID, Resource} from '@/simpli'
import {ResponseSerialize} from '@/simpli'

import User from '@/model/User'

export default class UserResp extends Resource {
  readonly $endpoint: string = '/Crud/User{/id}'

  @ResponseSerialize(User)
  user: User = new User()

  get $id() {
    return this.user.$id
  }

  set $id(val: ID) {
    this.user.$id = val
  }
}

File: packages/@simpli/cli-server/generator/template/src/main/java/br/com/martinlabs/usecase/crud/response/UserResp.kt
package br.com.martinlabs.usecase.crud.response

import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty
import br.com.martinlabs.usecase.model.User

@ApiModel(value = "UserResp")
class UserResp {

    var user: User? = null
}
